pkg/cache/memcached/memcached.go

```diff
@@ -12,17 +15,38 @@ import (
 
 // cache is a Cacher implemented on top of Memcached.
 type cache struct {
-	*memcache.Client
+	client     *memcache.Client
 	expiration int32
+	sync.RWMutex
```

Comment:
I'm not a huge fan of exposing these, I've seen this cause problems in the past as the methods happen to be callable from the outside.

Reply:
ack :+1: